src/priceCalculation.ts

```diff
@@ -288,6 +288,7 @@ export async function placeOrders(
         [sellOrder.buyAmount],
         [sellOrder.sellAmount],
         [queueStartElement],
+        "0x00",
```

Comment:
Does it work if the calldata is empty?
```suggestion
        "0x",
```